cmd/utils/ont.go

```diff
@@ -621,6 +623,23 @@ func GetTxHeight(txHash string) (uint32, error) {
 	return height, nil
 }
 
+func GetShardChainConfig(shardID uint64, height uint64) (*vconfig.ChainConfig, error) {
+	data, ontErr := sendRpcRequest("getshardchainconfig", []interface{}{shardID, height})
+	if ontErr != nil {
+		switch ontErr.ErrorCode {
+		case ERROR_INVALID_PARAMS:
```

Comment:
if switch one case, there could use ```if``` to replace

Reply:
ok